Assets/MixedRealityToolkit/_Core/DataProviders/SpatialObservers/Profiles/BaseMixedRealitySpatialMeshObserverProfile.cs

```diff
@@ -1,13 +1,24 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using Microsoft.MixedReality.Toolkit.Core.Attributes;
 using Microsoft.MixedReality.Toolkit.Core.Definitions.SpatialAwarenessSystem;
 using UnityEngine;
 
 namespace Microsoft.MixedReality.Toolkit.Core.DataProviders.SpatialObservers
 {
     public abstract class BaseMixedRealitySpatialMeshObserverProfile : BaseMixedRealitySpatialObserverProfile
     {
+        [PhysicsLayer]
+        [SerializeField]
+        [Tooltip("The optional mesh Physics layer override of the base PhysicsLayer")]
+        private int meshPhysicsLayerOverride = -1;
```

Comment:
Please add a similar property to the SurfaceObserver profile. Thanks,

Reply:
Done :)